addon/-private/diff-route-info.js

```diff
@@ -128,6 +130,46 @@ if (gte('3.6.0')) {
     return routes;
   }
 
+  // This should be invoked if there are no results for queryparams diff(qpsDiff) due to overlapping logic
+  pathsRefresh = function(from, to, intent) {
```

Comment:
Can you add some unit tests for this? See https://github.com/nickiaconis/ember-prefetch/blob/master/tests/unit/util/diff-route-infos-test.js for examples with the other utility methods.

Reply:
Added unit tests for pathsRefresh